apis/postgresql/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
The CI error is probably because of this line, the go version needs updating, I've got a PR for this here: https://github.com/crossplane-contrib/provider-sql/pull/67/files

Reply:
Yes, most likely. Should I remove the line, or should we wait for that PR to get merged?

Reply:
#67 has been merged, but it looks like the test is still failing. @petteja can you take a look?

Reply:
This branch now just needs a rebase to pull the new github actions (and probably resolve merge conflicts from other changes).

Reply:
Merged in changes from master

Reply:
Ran `make reviewable test e2e` on a different PC and it worked just fine. :man_shrugging: